chpl/chpl-resources/src/main/resources/errors.properties

```
@@ -1019,3 +1019,17 @@ subscription.objectTypeIdRequired=An object type for the subscription is require
 subscription.objectTypeIdInvalid=The provided object type ID '%s' is not valid.
 subscription.objectIdRequired=A subscribed object ID is required.
 subscription.objectIdInvalid=The subscribed object ID '%s' is not valid.
+
+criteriaAttribute.edit.emptyValue=Value is required.
+criteriaAttribute.edit.emptyRegulatoryTextCitation=Regulatory Text Citation is required.
+criteriaAttribute.edit.noCriteria=A %s requires at least one Criterion associated with it.
+criteriaAttribute.edit.duplicate=Regulatory Text Citation and Value matches an existing %s.
+criteriaAttribute.edit.deletedCriteria.listingsExist=This %s and Criteria %s is attested to by %s Listing%s
```

I don't like having periods at the ends of things that often show up as bulleted lists, but it seems we've gone down that path
```suggestion
criteriaAttribute.edit.deletedCriteria.listingsExist=This %s and Criteria %s is attested to by %s Listing%s.
```